examples/evaluators/golang/simple/main.go

```diff
@@ -26,41 +26,52 @@ package main
 
 import (
 	"context"
-	"fmt"
-	"log"
-	"os"
+	"flag"
 	"strings"
 	"time"
 
 	"github.com/GoogleCloudPlatform/open-match/config"
 	om_messages "github.com/GoogleCloudPlatform/open-match/internal/pb"
+	"github.com/GoogleCloudPlatform/open-match/internal/set"
 	redishelpers "github.com/GoogleCloudPlatform/open-match/internal/statestorage/redis"
+	"github.com/GoogleCloudPlatform/open-match/internal/statestorage/redis/ignorelist"
 	"github.com/GoogleCloudPlatform/open-match/internal/statestorage/redis/redispb"
 	"github.com/gobs/pretty"
 	"github.com/gomodule/redigo/redis"
+	log "github.com/sirupsen/logrus"
 	"github.com/spf13/viper"
 )
 
+var (
+	lgr    *log.Logger
+	cfg    *viper.Viper
+	err    error
```

Comment:
Having err as a global is a major pitfall because the err can be updated anywhere and give unpredictable results. This is particularly nasty with goroutines.

Reply:
fixed